generators/server/templates/build.gradle.ejs

```diff
@@ -96,6 +98,26 @@ eclipse {
     }
 }
 
+// See https://virgo47.wordpress.com/2018/09/14/classpath-too-long-with-spring-boot-and-gradle/ for details
+// https://github.com/jhipster/generator-jhipster/issues/9713
+if (OperatingSystem.current().isWindows()) {
+    task classpathJar(type: Jar) {
+        inputs.files sourceSets.main.runtimeClasspath
+
+        archiveName = "runboot-classpath.jar"
+        doFirst {
+            manifest {
+              def classpath = sourceSets.test.runtimeClasspath.files
```

Comment:
minor indent: should be 4 spaces for tab.
missing 2 spaces here :p

Reply:
Eagleyes :eyes: :+1:  Fixed.